src/Microsoft.ML.Parquet/ParquetLoader.cs

```diff
@@ -499,21 +499,21 @@ private Delegate CreateGetterDelegate(int col)
                     case DataType.Byte:
                         return CreateGetterDelegateCore<byte, byte>(col, _parquetConversions.Conv);
                     case DataType.SignedByte:
-                        return CreateGetterDelegateCore<sbyte?, DvInt1>(col, _parquetConversions.Conv);
```

Comment:
>sbyte?, [](start = 56, length = 7)

Is removing this right? The way I read this code is, it is listing the types that are supported in a Parquet file, and telling how to convert them into typical dataview types. So: mapping `float?` into `float`, and so forth. I think the left type parameter *must* remain the same since that comes from Parquet, not us -- you'll have to figure out what to do with those nullable values in that case.

Reply:
I guess this means that there is no test coverage for integers in parquet files? Perhaps that ought to be testsed.

---
In reply to: [212049581](https://github.com/dotnet/machinelearning/pull/683#discussion_r212049581) [](ancestors = 212049581)

Reply:
I spoke with parquet folks and we agreed to throw an exception even though its not desirable in a distributed environment. 

---
In reply to: [212049764](https://github.com/dotnet/machinelearning/pull/683#discussion_r212049764) [](ancestors = 212049764,212049581)